website/blog/missing-pieces-of-design-first.md

```diff
@@ -0,0 +1,91 @@
+
+## Evidence: did we build the change correctly?
+
+A Design First approach requires verifying we built the thing we designed. The design might be good, it may go through a fruitful discussion before getting to development, and it may meet all the quality requirements for APIs. But if we can't determine whether or not our code aligns with our design, we risk deploying the wrong thing and publishing an API reference that doesn't match the deployed API.
+
+Without an evidence-based workflow, we're crossing our fingers and hoping everything matches up, or we're hoping someone catches any unforeseen missteps when in review.
+
+For us to gather evidence, we need to look at the behavior of our API while it's running and see if it matches up with the API specification. This is ensuring the quality of our design makes it through to the implementation and gets into the hands of the API consumers.
+
+It's common for workflows to encourage hand-written tests to show evidence. Without a tool that can compare traffic with the proposed API specification, it's up to the person writing the tests to codify the API changes by hand.
+
+## Relationships: do we have open communication with consumers?
+
+Design First doesn't work if we never talk to the people who use the API. The goal of Design First is to gather people together who are interested in the value of the API. This includes the API consumers.
+
+Are people able to share the struggles encountered while using the API? Can consumers provide outside feedback on ways to improve the API? Does the team share upcoming changes with consumers so they can better prepare themselves rather than learn in production?
+
+API consumers have to be and informed part of the team. They need open channels that encourage discussion. They need insights into changes before they happen. This requires cultivating relationships between the API team and the consumers.
+
+Unfortunately, the API consumer is left out of workflows today. The burden is on them to keep up with API changes, and many times they find out about change when it's too late.
+
+## Putting these ideas to practice
+
+If you're trying to garner adoption in your company around a Design First process, consider these other areas that are an important part of the approach. It's easy to pick Design First as a practice without considering the full experience around how people build APIs.
+
+You can find ways to help people get more visibility into their API changes, collaborate around the changes, and ensure each change meets the quality standards of the company—all without requiring teams to abandon their existing workflows. You can help teams transition to a better way to build APIs in their own context.
+
+In a future post, we'll look at the ways Optic tries to help in all these areas to encourage a collaborative design and development experience for the entire team.
```

Comment:
It might fit in a few different places, but perhaps now is not the right time to talk about it: there's also the additional layer of understanding how each consumer uses the API, aka Optic's API analytics, to go deeper than just conformance.

Reply:
Would you see this as another section in the post? I've tried to fit this into the existing ones, and while it's related, it doesn't feel like it fits directly. It's almost like there's an "observability" component that lets teams analyze and understand the way people use the API to help inform the design.

I think this is a good idea, and it's something that isn't in the Design First discussion—I even left it out here myself :)

Do you think it's worth adding? If so, how do you feel about the length of the post since this will add a few paragraphs?

Reply:
I think it can come later :) definitely worth adding but can be part of the optic or another article